File: _src/05/Chapter5.CSharp/Maps/CompanyMap.cs
using System.Data.Entity.ModelConfiguration;
using Chapter5.CSharp.Models;

namespace Chapter5.CSharp.Maps
{
    public class CompanyMap : EntityTypeConfiguration<Company>
    {
        public CompanyMap()
        {
            Property(p => p.CompanyName)
                .HasMaxLength(30)
                .IsRequired();
        }
    }
}

File: _src/05/Chapter5.VB/Maps/CompanyMap.vb
Imports System.Data.Entity.ModelConfiguration
Imports Chapter5.VB.Models

Namespace Maps
    Public Class CompanyMap
        Inherits EntityTypeConfiguration(Of Company)
        Public Sub New()
            Me.Property(Function(p) p.CompanyName) _
                .HasMaxLength(30) _
                .IsRequired()
        End Sub
    End Class
End NameSpace